cg/apps/scout/scoutapi.py

```diff
@@ -302,25 +302,25 @@ def upload_omics_sample_id(
     def upload_rna_fraser_outrider(
         self,
         case_id: str,
-        fraser_file_path: str,
-        outrider_file_path: str,
+        fraser_file_path: str | None,
+        outrider_file_path: str | None,
         customer_case_name: str,
         cust_id: str,
     ) -> None:
         """Load a rna fraser file into a case in the database."""
-
         upload_command: list[str] = [
             "update",
             "case",
             "-n",
             customer_case_name,
             "-i",
             cust_id,
-            "--fraser",
-            fraser_file_path,
-            "--outrider",
-            outrider_file_path,
         ]
+        if fraser_file_path:
+            upload_command.extend(["--fraser", fraser_file_path])
+
+        if outrider_file_path:
+            upload_command.extend(["--outrider", outrider_file_path])
         try:
             LOG.info(
                 f"Uploading rna fraser file {fraser_file_path} and outrider file {outrider_file_path} to case {case_id} with command {upload_command}"
```

Comment:
```suggestion
            LOG.info(
                f"Uploading rna fraser file {fraser_file_path} and outrider file {outrider_file_path} to case {case_id} with command {upload_command}"
```

Would be a bit confusing if outrider_file_path is None